packages/api/src/api/helix/moderation/HelixBanUser.ts

```diff
@@ -21,10 +21,10 @@ export class HelixBanUser extends DataObject<HelixBanUserData> {
 	}
 
 	/**
-	 * The UTC date and time (in RFC3339 format) that the timeout will end. Is null if the user was banned instead of put in a timeout.
+	 * The date and time that the timeout will end. Is `null` if the user was banned instead of put in a timeout.
 	 */
-	get endTime(): string {
-		return this[rawDataSymbol].end_time;
+	get endTime(): Date | null {
```

Comment:
All date properties should end with `Date` in the same, even if they also include a time (most of them do).

Reply:
Fixed. You might consider adding things like this to the `CONTRIBUTING.md` in the root of the project to reduce rework for new contributors. Good, well thought out coding standards/conventions make for better contributions.